dotnet/src/AutoGen.Core/Message/ImageMessage.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// ImageMessage.cs
+
+namespace AutoGen.Core;
+
+public class ImageMessage : IMessage
+{
+    public ImageMessage(Role role, string url, string? from = null)
+    {
+        this.Role = role;
+        this.From = from;
+        this.Url = url;
```

Comment:
Is this the URL to the image for this message? Would a message contain more than one images?

Reply:
Yes, and for multi-image situations, `MultiModalMessage` is the message to use.